File: LottoPlayer/app/src/main/java/com/im/daeseong/lottoplayer/LottoTopRecyclerAdapter.java
package com.im.daeseong.lottoplayer;

import android.content.Context;
import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.text.Html;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.im.daeseong.lottoplayer.Database.LottoTop;
import com.im.daeseong.lottoplayer.Util.Lottoutil;
import com.im.daeseong.lottoplayer.Util.RoundView;

import java.util.List;

public class LottoTopRecyclerAdapter extends RecyclerView.Adapter<LottoTopRecyclerAdapter.ViewHolder> {

    private static final String TAG = LottoRecyclerAdapter.class.getSimpleName();

    private Context context;
    private List<LottoTop> LottoTop_list;

    public LottoTopRecyclerAdapter(Context context, List<LottoTop> LottoTop_list){
        this.context = context;
        this.LottoTop_list = LottoTop_list;
    }

    @NonNull
    @Override
    public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View view = LayoutInflater.from(context).inflate(R.layout.lotto_top_item, parent, false);
        return new ViewHolder(view);
    }

    @Override
    public void onBindViewHolder(@NonNull ViewHolder holder, int position) {

        //String stv1 = String.format("<font color=\"#000000\">숫자 </font><font color=\"#ff9900\">%d</font><font color=\"#000000\"> 는</font>", LottoTop_list.get(position).getNumber());
        //holder.tv1.setText(Html.fromHtml(stv1));
        String stv1 = String.format("%s", LottoTop_list.get(position).getNumber());
        holder.tv1.setTitleText(stv1);//holder.tv1.setText(stv1);
        holder.tv1.setBackgroundColor(Lottoutil.getLottoColor(LottoTop_list.get(position).getNumber()));//holder.tv1.setTextColor(Lottoutil.getLottoColor(lottoList.get(position).getPart1()));

        String stv2 = String.format("<font color=\"#ff9900\">%d</font><font color=\"#000000\"> 회 당첨 되었습니다.</font>", LottoTop_list.get(position).getCount());
        holder.tv2.setText(Html.fromHtml(stv2));
    }

    @Override
    public int getItemCount() {
        return LottoTop_list.size();
    }

    public class ViewHolder extends RecyclerView.ViewHolder {

        public final RoundView tv1;//public final TextView tv1;
        public final TextView tv2;

        public ViewHolder(View view){
            super(view);

            tv1 = (RoundView)view.findViewById(R.id.tv1);//(TextView)view.findViewById(R.id.tv1);
            tv2 = (TextView)view.findViewById(R.id.tv2);
        }
    }

}

File: LottoPlayer(kotlin)/app/src/main/java/com/daeseong/lottoplayer/LottoTopRecyclerAdapter.kt
package com.daeseong.lottoplayer

import android.content.Context
import android.os.Build
import android.text.Html
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.daeseong.lottoplayer.Database.LottoTop
import com.daeseong.lottoplayer.Util.Lottoutil.getLottoColor
import com.daeseong.lottoplayer.Util.RoundView

class LottoTopRecyclerAdapter(private val context: Context, private val lottoTopList: List<LottoTop>) : RecyclerView.Adapter<LottoTopRecyclerAdapter.ViewHolder>() {

    inner class ViewHolder(view: View) : RecyclerView.ViewHolder(view) {
        val tv1: RoundView = view.findViewById(R.id.tv1)
        val tv2: TextView = view.findViewById(R.id.tv2)
    }

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
        val view: View = LayoutInflater.from(context).inflate(R.layout.lotto_top_item, parent, false)
        return ViewHolder(view)
    }

    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
        val currentItem = lottoTopList[position]

        holder.tv1.apply {
            setTitleText(currentItem.number.toString())
            setBackgroundColor(getLottoColor(currentItem.number))
        }

        val stv2 = String.format("<font color=\"#ff9900\">%d</font><font color=\"#000000\"> 회 당첨 되었습니다.</font>", currentItem.count)

        holder.tv2.text = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            Html.fromHtml(stv2, Html.FROM_HTML_MODE_LEGACY)
        } else {
            Html.fromHtml(stv2)
        }
    }

    override fun getItemCount(): Int = lottoTopList.size
}
